src/player.cpp

```diff
@@ -7368,12 +7368,14 @@ item::reload_option player::select_ammo( const item &base, std::vector<item::rel
             const std::function<std::string( int )> draw_row;
             int last_key;
             const int default_to;
+            const bool can_partial_reload;
 
             reload_callback( std::vector<item::reload_option> &_opts,
                              std::function<std::string( int )> _draw_row,
-                             int _last_key, int _default_to ) :
+                             int _last_key, int _default_to, bool preload ) :
                            opts( _opts ), draw_row( _draw_row ),
-                           last_key( _last_key ), default_to( _default_to )
+                           last_key( _last_key ), default_to( _default_to ),
+                           can_partial_reload( preload )
```

Comment:
Why is the parameter named `preload`? What has this to do with partial reload?

Reply:
thanks. i probably shouldnt shorten variable names like that. bad habit.